### ISSUE TITLE
meson 0.54.0 test output is needlessly precise

### ISSUE THREAD

**Issue by ddevault** [mesonbuild/meson#6884]
Since meson 0.54.0, I see test results like this:

```
 1/20 parse/arithm                            OK      0.02118539810180664 s
 2/20 parse/basic                             OK      0.017963886260986328 s
 3/20 parse/cast                              OK      0.019874095916748047 s
 4/20 parse/cond                              OK      0.0275113582611084 s
 5/20 parse/fndecl                            OK      0.01894211769104004 s
 6/20 parse/variadic                          OK      0.02661752700805664 s
 7/20 compile/access                          OK      0.011080741882324219 s
 8/20 compile/arithm                          OK      0.009584188461303711 s
 9/20 compile/assign                          OK      0.006812095642089844 s
10/20 compile/conditionals                    OK      0.007668495178222656 s
11/20 compile/pointers                        OK      0.010450601577758789 s
12/20 failures/access_if_without_else         EXPECTEDFAIL0.43506908416748047 s
13/20 failures/assign_to_const                EXPECTEDFAIL0.4292886257171631 s
14/20 failures/dangling_parameter             EXPECTEDFAIL0.4297518730163574 s
15/20 failures/dangling_pointer               EXPECTEDFAIL0.39390134811401367 s
16/20 failures/declare_void                   EXPECTEDFAIL0.43333959579467773 s
17/20 failures/free_in_one_branch             UNEXPECTEDPASS0.01156926155090332 s
18/20 failures/pointer_reuse                  EXPECTEDFAIL0.37464451789855957 s
19/20 failures/variable_out_of_scope          EXPECTEDFAIL0.4337000846862793 s
20/20 failures/variable_rebinding             EXPECTEDFAIL0.4273262023925781 s
```

Rounding to two or three decimal points would be a lot easier to parse - this much precision is difficult to read and compare between tests, and unlikely to be consistent between test runs. For users who want this much precision, maybe a flag or environment variable would be suitable.

Additionally, it seems that `EXPECTEDFAIL` and `UNEXPECTEDPASS` are rubbing right up against the numbers - there ought to be a space here.

I also tested this on the latest meson from master (7828f2c9c2e7fb1cbeb1ee5bcaa415d0aa0eb622).

**Comment by mensinda** [comment]
This is fixed with #6885. Closing.

**Comment by ddevault** [comment]
Thanks!
